pkg/backup/client.go

```diff
@@ -815,3 +865,14 @@ func (bc *Client) CompleteMeta(backupSchemas *Schemas) error {
 	bc.backupMeta.Schemas = schemas
 	return nil
 }
+
+// CopyMetaFrom copies schema metadata directly from pending backupSchemas, without calculating checksum.
+// use this when user skip the checksum generating.
+func (bc *Client) CopyMetaFrom(backupSchemas *Schemas) {
```

Comment:
why do we need a deep copy here? i.e. why running

```go
bc.backupMeta.Schemas = backupSchemas.schemas
```

isn't sufficient?

Reply:
I'm not sure whether it's OK to just shallow copy(i.e. will the slice be unexpected changed somewhere?), the original version(`schemas.Start()` -> `schemas.finishTableChecksum()` -> `client.CompleteMeta()`) makes a deep copy of the slice, so I just followed it...

Reply:
BTW, I think it won't be costly since the number of schema is usually small... and this method is invoked just once per backup...